src/palace/manager/api/controller/patron_activity_history.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import uuid
+
+import flask
+from flask import Response
+
+from palace.manager.api.controller.circulation_manager import (
+    CirculationManagerController,
+)
+
+
+class PatronActivityHistoryController(CirculationManagerController):
+
+    def erase(self):
```

Comment:
Should have a type hint here: `-> Response:`

I've been trying to add any new code / controllers that we write to the pyproject.toml mypy config https://github.com/ThePalaceProject/circulation/blob/ebcddcab60487e7f9a763b27296c39aca04f2649/pyproject.toml#L58-L69

So they are strictly type checked from the beginning. Maybe this new controller can be added there.